src/templates/pages/server_status.html

```diff
@@ -2,8 +2,70 @@
 {% load staticfiles %}
 
 {% block extra_head %}
-    <script src="https://bernii.github.io/gauge.js/dist/gauge.min.js"></script>
 {% endblock %}
+
 {% block content %}
-    <server_status></server_status>
+    <div class="ui container">
+        <h1>Recent submissions (up to 250 or 2 days old)</h1>
+        <table class="ui table">
+            <thead>
+            <th>Competition</th>
+            <th>Submitter</th>
+            <th>Submitted at</th>
+            <th>Status</th>
+            </thead>
+            <tbody>
+                {% for sub in submissions %}
+                <tr>
+                    <td>{{ sub.phase.competition }}</td>
+                    <td>{{ sub.owner.username }}</td>
+                    <td>{{ sub.created_when|timesince }} ago</td>
+                    <td>{{ sub.status }}</td>
+                </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+
+        <h1>Monitor queues</h1>
+        <div id="external_monitors" class="ui two column grid">
+            <div class="column">
+                <div class="ui fluid card">
+                    <a class="image" href="#">
+                        <img class="ui large image" src="/static/img/RabbitMQ.png">
+                    </a>
+                    <div class="content">
+                        <a class="header" href="//{{RABBITMQ_MANAGEMENT_URL}}" target="_blank">RabbitMQ</a>
+                        <div class="meta">
+                            <a href="#">
```

Comment:
Should this be an `a` tag? or should it have an `href`?

Reply:
Oh.. didn't even notice these, I just removed them? ✔️